src/components/Articles.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { Row } from 'reactstrap';
+import PropTypes from 'prop-types';
+import smartTruncate from 'smart-truncate';
+
+class Articles extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      articles: [],
+    };
+  }
+
+  componentDidMount() {
+    const { fetchPersonalArticles } = this.props;
+    fetchPersonalArticles();
+  }
+
+  componentWillReceiveProps(newProps) {
+    this.setState({
+      articles: newProps.personalArticles.personalArticles.Articles,
+    });
+  }
+
+  render() {
+    const { articles } = this.state;
+    const articleItems = articles.length > 0 ? (
+      articles.map(article => (
+        <div className="article-card" key={article.id}>
+          <div className="color-overlay">
+            <div className="article-content">
+              <div className="article-header">
+                <h3 className="article-title">{article.title}</h3>
+                <h3 className="article-info">Crptocurrency Finance</h3>
+              </div>
+              <p className="article-desc">{smartTruncate(article.body, 100)}</p>
+              <button type="submit" className="btn-two">Read more</button>
+            </div>
+          </div>
+        </div>
```

Comment:
@BrianSerem, place all this JSX in a file of its own, then map over its component parsing in props.